lib/matplotlib/figure.py

```diff
@@ -1087,6 +1087,16 @@ def legend(self, *args, **kwargs):
         ----------------
         %(_legend_kw_doc)s
 
+        If a figure is using the constrained layout manager, the string codes
```

Comment:
Due to limitations in numpydoc you cannot add a paragraph to a parameters section. This does not render correctly: https://output.circle-artifacts.com/output/job/1014b593-20dc-47c6-a1f5-00a6d78f9818/artifacts/0/doc/build/html/api/figure_api.html#matplotlib.figure.Figure.legend

More generally, this would better be placed in the `loc` parameter section. This is already a mix of Axes / Figure legend information. The simple solution is to move this in there and say "For figure legends ...". Possibly slightly better but more work is to pull the `loc` description out of `_legend_kw_doc` and instead provide individual descriptions for figure and axes legends; i.e.

```
Other Parameters
----------------
loc : 
    description
%(_legend_kw_doc)s
```